lib/cloudregister/version.py

```diff
@@ -0,0 +1 @@
+__VERSION__ = '10.1.2'
```

Comment:
Let's keep the current implementation with `VERSION` and `read`

Reply:
OK, done
out of curiosity, is there a reason to keep it like that ?
other than that it is the current implementation